### ISSUE TITLE
Documentation for the new language plugins

### ISSUE THREAD

**Issue by nylonee** [nylonee/hangupsbot#94]
It'd be nice to have some documentation on how the new language plugins work for both syncrooms and same-room translations.

**Comment by endofline** [comment]
From my side, I will update docs after #79 is completed. Feel free to either add into README, or the wiki whichever is more convenient. Personally, my doc roadmap (as the final part of #79) will be:
* updating `__doc__` for publicly available commands
* general update of README and wiki, including which parts to move over (if any)

**Comment by endofline** [comment]
https://github.com/nylonee/hangupsbot/wiki/Chat-Relays-with-Syncouts + README being updated ( a6e30df, 8940d6e)

**Comment by endofline** [comment]
meh, restructuring the README is a real PITA - most likely will rewrite the entire thing

**Comment by endofline** [comment]
Please note the large change in the README file, currently residing here: https://github.com/nylonee/hangupsbot/blob/framework-plugin-admincontrol/README.md

**Comment by endofline** [comment]
The changes has been staged

**Comment by endofline** [comment]
released as part of https://github.com/nylonee/hangupsbot/releases/tag/2.1.1
